.coveragerc

```diff
@@ -12,6 +12,7 @@ omit =
     *aimsun/run.py
     *aimsun/api.py
     *aimsun/generate.py
+    *aimsun/load.py
```

Comment:
Good call on this. But we should begin thinking about how we are going to start testing aimsun features. I've been thinking about generating dummy modules to replace the aimsun module's in tests, but we may need to figure out more clever ways as well.

Reply:
I'll look into it! By the way, why is `*aimsun/api.py` in this file if we have a `test_aimsun_api.py` file?

Reply:
mm, ya, I think that script doesn't actually cover the full API, but uses a dummy instead, and just checks stuff like the aimsun objects we created. Something to look to in the future as well.